README.md

```diff
@@ -210,7 +229,8 @@ Google Cloud Resource Manager (Alpha)
 #### Preview
 
 Here is a code snippet showing a simple usage example. Note that you must supply Google SDK credentials for this service, not other forms of authentication listed in the [Authentication section](#authentication).
-
+Complete source code can be found at
```

Comment:
should we check if `myProject` is `null`?
Also, I would rename myProject to just project.